code-challenges.js

```diff
@@ -4,65 +4,154 @@
+    
+        console.log(boilingTemp(42))
+        console.log(boilingTemp(350))
+        console.log(boilingTemp(212))
+        }
+
+    // No results returned - Not sure why??
+
+
+      
+   
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
 
+// Create code that combines two arrays
+// input: two test variables const padres1984WorldSeriesRuns and const padres1998WorldSeriesRuns 
+// Use the .length method and combine the two test variables using the + sign
+// output: returns the length of the two variables
+
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
+// Actual output: 9
+
+console.log (padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
+
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
 
+// Create code to reverse letters in a string
+// Input: const currentCohort
+// Need to create perform three actions in sequential order
+// 1. Need to split the text string into individual characters
+// 2. Need to reverse the order of the characters
+// 3. Need to join all the character in the reversed order     
+// Output: "Golf 2022 will be reversed"
+
+//const currentCohort = "Golf 2022"
+
 const currentCohort = "Golf 2022"
+const splitString = (currentCohort.split(""))
+const reverseString = (splitString.reverse())
+const joinArray = (reverseString.join(""))
+console.log (joinArray)
+
 // Expected output: "2202 floG"
+// Actual output "2202 floG"
+
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Describe your process:
 
+// Create code to index a value from an array
+// Input: givenValue1 and givenValue2 
+//  Use .lastIndexOf method of each given value
+// Output: last index of the given values
+
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 const givenValue1 = 42
+console.log (myNumbers.lastIndexOf(42))
 // Expected output: 7
+// Actual output: 7
 
 const givenValue2 = 10
+console.log (myNumbers.lastIndexOf(10))
+
 // Expected output: 8
+// Actual output: 8
 
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Describe your process:
 
+// Create code that sorts numbers from largest to smallest in an array
+// Input: test variables sanDiegoSummerTemperatures and sanDiegoWinterTemperatures
+//   Use .sort method for each test variable to get smallest to largest
+//   Use .reverse method for each test variable to reverse the order to largest to smallest
+
+
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
+sanDiegoSummerTemperatures.sort((a, b) => a - b)
```

Comment:
I love using the sort method for this problem!  Make sure to add into your pseudocode what is happening within the sort method and why that is needed.